posthog/settings/temporal.py

```diff
@@ -15,6 +15,7 @@
 BATCH_EXPORT_SNOWFLAKE_UPLOAD_CHUNK_SIZE_BYTES = 1024 * 1024 * 100  # 100MB
 BATCH_EXPORT_POSTGRES_UPLOAD_CHUNK_SIZE_BYTES = 1024 * 1024 * 50  # 50MB
 BATCH_EXPORT_BIGQUERY_UPLOAD_CHUNK_SIZE_BYTES = 1024 * 1024 * 100  # 100MB
-
+BATCH_EXPORT_HTTP_UPLOAD_CHUNK_SIZE_BYTES = 1024 * 1024 * 10  # 10MB
+BATCH_EXPORT_HTTP_BATCH_SIZE = 1000
```

Comment:
Xavier said: "Maximum request body is 20MiB uncompressed, might we worth stating at 1000 ev/batch and making it configurable."

We have both limits because super large events could easily go over 20MB before hitting a batch of 1000.

Reply:
Eventually this is something that should be configurable, fine to have both limits to start off with just us for migration purposes.